### ISSUE TITLE
Window no longer rezieable

### ISSUE THREAD

**Issue by martinlest** [albar965/littlenavmap#241]
Was just notified of an update, so installed it. Handles at windows edges no longer resize the LNM window, nothing happens when you drag - so gone back to previous version. Please fix. Thanks.

**Comment by martinlest** [comment]
"rezieable"! ???? Resizeable, of course. Typos getting worse and worse...

**Comment by albar965** [comment]
Windows? Multi monitor? I remember you use X-Plane.

First try `Window` -> `Reset window layout`
If that does not help: Can you temporary rename the file 
```
C:\Users\YOURUSERNAME\AppData\Roaming\ABarthel\little_navmap.ini
``` 
to check if something went wrong when reading the old settings?

Have not seen this yet.
Alex

**Comment by martinlest** [comment]
Hi Alex,
Did all that. I can resize now, but the minimum size of the window is bigger than the previous version - it won't drag any smaller after a certain point, so it no longer fits with my other XP apps on the desktop -  it overlaps other windows. Guess I can juggle my various windows around to make room for it!
Thanks,
Martin

**Comment by albar965** [comment]
Hi Martin,
the airport search window has a few buttons that keep the window large. I have to rearrange some of them. Use the "hamburger"-button ![Menu](https://albar965.gitbooks.io/little-navmap-user-manual/content/images/icons/menubutton.png) to disable all unneeded search options. Then you can increase the size of the map window again.
Alex

**Comment by albar965** [comment]
BTW: This is more a problem with the translated user interface. Do you have a German or French system?
Alex

**Comment by albar965** [comment]
I'll rearrange the buttons in the next minor update. I'll probably even hide all the fancy searches by default.
Alex

**Comment by albar965** [comment]
Fixed with 2.0.3.
